packages/core/types/src/types/core/attributes/blocks.ts

```diff
@@ -56,6 +56,7 @@ export interface ListBlockNode extends BaseNode {
   type: 'list';
   format: 'ordered' | 'unordered';
   children: (ListItemInlineNode | ListBlockNode)[];
+  listIndentLevel?: number;
```

Comment:
since we're already within the context of a list block, how about just indentLevel?